packages/storybook/src/stories/grid-pagination.stories.tsx

```diff
@@ -130,9 +130,9 @@ export function PaginationApiTests() {
             Pagination: ({ state }) => (
               <Pagination
                 className="my-custom-pagination"
-                page={state.pagination.page}
+                page={state.pagination.page + 1}
```

Comment:
Why do we need it here? Is it to load the second page?

Reply:
No it's for the component. As it's 0 indexed, we need to add 1 to convert it to the page number from the index